src/main/java/org/broadinstitute/hellbender/utils/variant/writers/HomRefBlock.java

```diff
@@ -94,15 +94,20 @@ private Genotype createHomRefGenotype(String sampleName) {
     }
 
     /**
-     * Add information from this Genotype to this band
-     * @param genotype a non-null Genotype with GQ and DP attributes
+     * Add information from this Genotype to this band.
+     *
+     * Treats GQ values > 99 as 99.
+     *
+     * @param pos Current genomic position. Must be 1 base after the previous position
+     * @param genotype A non-null Genotype with GQ and DP attributes
      */
     public void add(final int pos, final Genotype genotype) {
         Utils.nonNull(genotype, "genotype cannot be null");
         if ( ! genotype.hasPL() ) { throw new IllegalArgumentException("genotype must have PL field");}
         if ( pos != end + 1 ) { throw new IllegalArgumentException("adding genotype at pos " + pos + " isn't contiguous with previous end " + end); }
         if ( genotype.getPloidy() != ploidy) { throw new IllegalArgumentException("cannot add a genotype with a different ploidy: " + genotype.getPloidy() + " != " + ploidy); }
-        if ( !withinBounds(genotype.getGQ())) {
+        // Make sure the GQ is within the bounds of this band. Treat GQs > 99 as 99.
+        if ( !withinBounds(Math.min(genotype.getGQ(), VCFConstants.MAX_GENOTYPE_QUAL))) {
             throw new IllegalArgumentException("cannot add a genotype with GQ=" + genotype.getGQ() + " because it's not within bounds ["
                     + this.getGQLowerBound() + ',' + this.getGQUpperBound() + ')');
```

Comment:
I think we need to make the constructor cap the maximum upper bound at 99 as well.  Right now it's possible to make one with a higher upper bound and then this downconversion will cause issues.